src/components/revisions/unrevised-entity.tsx

```diff
@@ -159,10 +158,12 @@ export function UnrevisedEntity({ entity, isOwn }: UnrevisedEntityProps) {
   }
 }
 
-const Td: CompBaseProps<{
-  centered?: boolean
-  className?: string
-}> = ({ children, centered, className }) => (
+const Td: FunctionComponent<
```

Comment:
Why not `function Td({ children, centered, className }: React.PropsWithChildren<...>) {... }`